core/scc/inprocstream.go

```diff
@@ -21,7 +21,7 @@ func (e SendPanicFailure) Error() string {
 	return fmt.Sprintf("send failure %s", string(e))
 }
 
-// PeerChaincodeStream interface for stream between Peer and chaincode instance.
+// InProcStream interface for stream between Peer and chaincode instance.
```

Comment:
```suggestion
// inProcStream interface for stream between Peer and chaincode instance.
```
For name consistency, it should start with a lowercase letter.

Reply:
Thanks. Modified